tests/kernel/mem_protect/userspace/testcase.yaml

```diff
@@ -0,0 +1,4 @@
+tests:
+  - test:
+      filter: CONFIG_CPU_HAS_MPU or CONFIG_X86_MMU
+      tags: core security ignore_faults
```

Comment:
Should instead be filtered on CONFIG_ARCH_HAS_USERSPACE and have the userspace tag, just drop and use mine.